<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNetPush</class>
 <widget class="QWidget" name="frmNetPush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labTip">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frameVideo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frameSlider">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="labPosition">
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labSlash">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labDuration">
        <property name="text">
         <string>00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Slider" name="sliderPosition">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSoundMuted">
        <property name="text">
         <string>静音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckPlayPause">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabBase">
         <attribute name="title">
          <string>基本参数</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnStart">
              <property name="text">
               <string>启动服务</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnPreview">
              <property name="text">
               <string>网页预览</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnRemove">
              <property name="text">
               <string>移除选中</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btnClear">
              <property name="text">
               <string>清空所有</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnAddFile">
              <property name="text">
               <string>添加文件</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="btnAddPath">
              <property name="text">
               <string>添加目录</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="btnAddUrl">
              <property name="text">
               <string>添加地址</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="btnClose">
              <property name="text">
               <string>关闭画面</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="labPushType">
               <property name="text">
                <string>类型</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cboxPushType">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labPushMode">
               <property name="text">
                <string>模式</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cboxPushMode">
               <item>
                <property name="text">
                 <string>rtmp</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>rtsp</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labPushHost">
               <property name="text">
                <string>网卡</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="cboxPushHost">
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QLineEdit" name="txtPushUrl"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabOther">
         <attribute name="title">
          <string>其他参数</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QComboBox" name="cboxFlagType">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>标识递增</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>指定标识</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>随机标识</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cboxCopyNumber"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labEncodeVideoScale">
            <property name="text">
             <string>缩放</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labFlagType">
            <property name="text">
             <string>策略</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labCopyNumber">
            <property name="text">
             <string>复制</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cboxEncodeVideoRatio"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labEncodeVideo">
            <property name="text">
             <string>视频</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cboxPreviewType">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>hls</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>flv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ws-flv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>webrtc</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labPreviewType">
            <property name="text">
             <string>预览</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labPushFlag">
            <property name="text">
             <string>标识</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labEncodeVideoRatio">
            <property name="text">
             <string>质量</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="cboxEncodeVideoScale">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtPushFlag"/>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cboxEncodeVideo"/>
          </item>
          <item row="7" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="ckCheckB">
              <property name="text">
               <string>校验B帧</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ckHideUser">
              <property name="text">
               <string>隐藏用户</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtRtspUrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtRtmpUrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtHlsUrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtWebRtcUrl">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckVideoVisible">
        <property name="text">
         <string>视频预览窗口</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cboxPushType</tabstop>
  <tabstop>cboxPushMode</tabstop>
  <tabstop>cboxPushHost</tabstop>
  <tabstop>txtPushUrl</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnPreview</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnAddFile</tabstop>
  <tabstop>btnAddPath</tabstop>
  <tabstop>btnAddUrl</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>txtRtspUrl</tabstop>
  <tabstop>txtRtmpUrl</tabstop>
  <tabstop>txtHlsUrl</tabstop>
  <tabstop>txtWebRtcUrl</tabstop>
  <tabstop>ckVideoVisible</tabstop>
  <tabstop>sliderPosition</tabstop>
  <tabstop>ckSoundMuted</tabstop>
  <tabstop>ckPlayPause</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>cboxFlagType</tabstop>
  <tabstop>txtPushFlag</tabstop>
  <tabstop>cboxPreviewType</tabstop>
  <tabstop>cboxCopyNumber</tabstop>
  <tabstop>cboxEncodeVideo</tabstop>
  <tabstop>cboxEncodeVideoRatio</tabstop>
  <tabstop>cboxEncodeVideoScale</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
